### PR TITLE
Added duration support to the Merten Roller Shutter drive

### PR DESCRIPTION
**Pull request by 7h0mas-R**
I added the function to support setting the duration of the Merten Roller Shutter driver following the request [on this post](https://community.home-assistant.io/t/hab-mal-ne-frage-bzgl-deiner-merten-rolladen-mqtt-losung/363990/3?u=7h0mas-r)
Tested it and it is working for me (function was also already implemented for the `PUCK/SHUTTER/1`device.

### CODE REVIEW

File: devices/schneider_electric.js [hunk 1]
```diff
@@ -439,13 +439,14 @@ module.exports = [
...
         fromZigbee: [fz.cover_position_tilt, fz.command_cover_close, fz.command_cover_open, fz.command_cover_stop],
-        toZigbee: [tz.cover_position_tilt, tz.cover_state],
-        exposes: [e.cover_position()],
+        toZigbee: [tz.cover_position_tilt, tz.cover_state, tzLocal.lift_duration],
+        exposes: [e.cover_position(), exposes.numeric('lift_duration', ea.STATE_SET).withUnit('seconds')
+            .withValueMin(0).withValueMax(300).withDescription('Duration of lift')],
         meta: {coverInverted: true},
         configure: async (device, coordinatorEndpoint, logger) => {
-            const endpoint = device.getEndpoint(1) || device.getEndpoint(5);
+            const endpoint = device.getEndpoint(5);
```
[the remark this answers is not in the document]
Comment (7h0mas-R) [reply devices/schneider_electric.js:449]: The duration setting was missing in my original implementation.
When I added it after receiving a Hint, I also realized, that endpoint 1 is not needed for the roller shutter function.
It is only used, when the controller is attached to a light switch insert. 
I removed it from my external Converter and did not notice anything missing.

Best regards

[the remark this answers is not in the document]
Comment (Koenkk) [reply devices/schneider_electric.js:449]: It was added due to https://github.com/Koenkk/zigbee2mqtt/issues/5561 so I believe it should stay

[the remark this answers is not in the document]
Comment (7h0mas-R) [reply devices/schneider_electric.js:449]: But in the thread referenced, you added Endpoint 5. I removed endpoint 1. 
As mentioned, I paired 3 of these devices yesterday without any functionality going missing.

But you can leave it in as well, does not do any harm from what I can see.
Want me to update the PR with the OR condition still in?

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**